styles/pymakr.less

```diff
@@ -370,6 +384,20 @@
     .pymakr-logo {
       height: 17px;
       padding-bottom: 2px;
+      border: 1px solid red;
+      fill: blue;
+      svg {
+        fill: red;
```

Comment:
Why we are using red and blue colors here, please?

Reply:
Oops